### ISSUE TITLE
JetBrains: Set up a Linux test environment with a UI on GCP

### ISSUE THREAD

**Issue by vdavid** [sourcegraph/sourcegraph#36706]
Clue: we don't need to set up a whole build environment, just a process to upload the built versions of the plugin.

**Comment by sourcegraph-bot-2** [comment]
Heads up @muratsu @jjinnii @ryankscott - the "team/integrations" label was applied to this issue.

**Comment by vdavid** [comment]
Done!

3-min Loom demo here: https://www.loom.com/share/447590223e6e4cb6af2a2666a9f17eb0 (same as for Windows)
Waiting for review from @philipp-spiess, if everything looks good, feel free to close this.

**Comment by philipp-spiess** [comment]
Notes here: https://docs.google.com/document/d/1ZvsSwc5oPi6Q4YVqy0DUcy-2FCGtswNxk5omxaIBld0/edit#heading=h.3ecr38rfbh7d

**Comment by philipp-spiess** [comment]
@vdavid  I’m having issues with this approach, specifically with the step of pasting the config command via SSH:

```
[0621/110740.078883:ERROR:host_config.cc(57)] Failed to read /home/philipp_spiess/.config/chrome-remote-desktop/host#b073bd424431c01ba317ca44b160506a.json
[0621/110741.751807:INFO:daemon_controller_delegate_linux.cc(97)] Created symlink /etc/systemd/system/multi-user.target.wants/chrome-remote-desktop@philipp_spiess.service → /lib/systemd/system/chrome-remote-desktop@.service.
```

Am I doing something wrong?

**Comment by vdavid** [comment]
Nope, that error message is very confusing, it took some time for me as well to realize this! You can actually go ahead to https://remotedesktop.google.com/access and connect.

**Comment by philipp-spiess** [comment]
OH WAIT THIS IS EXPECTED? 😮  I will add this to the docs. Yeah I can connect now.

The instance appears to be debian though (which has a very minimal UI that I am not comfortable with) and I can't find IntelliJ on this instance. Where do I find it?

**Comment by vdavid** [comment]
Yes, this is Debian, but that isn't the cause for the minimalistic UI. It's Xfce.
I couldn't get KDE installed, it failed for some reason. We can give Gnome or anything else a shot that's listed [here](https://cloud.google.com/architecture/chrome-desktop-remote-on-compute-engine#installing_an_x_windows_system_desktop_environment).
I think what happened is that for Linux, you have a separate user from mine, and I extracted IntelliJ in my home folder. You can `sudo` and go to my home and make it more public if you want! Or I can do that. Or you can install another instance. Not sure which is the best way to go.
I added a shortcut to it to my desktop, but apparently it's not _your_ desktop 🤷‍♂️
Do you need help with any of this, or will you give it a shot?

**Comment by vdavid** [comment]
Oh, and I've already added it to the docs when I wrote my response.
